testsuite/features/step_definitions/common_steps.rb

```diff
@@ -299,7 +304,7 @@
   unless out.include? 'ssh-push-tunnel'
     steps %(
       Given I am authorized as "testing" with password "testing"
-      And I follow this "#{host}" link
+      And I am on the Systems overview page of this "sle_client"
```

Comment:
No no, don't hardcore this step with "sle_client" as this might be used with other values in "host", keep using #{host} but ok to use this different step.

Reply:
Good thing you pointed that out, I had this same change in another PR that was actually already merged in some branches
I'll fix that asap